website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -110,138 +115,161 @@ Here is the current architecture of our application:
-├── express@4.13.4
-└── jade@1.11.0
-html-server  PROCESS ENDED
-html-server  EXIT CODE: 0
-  exo-setup  setup of 'html-server' finished
-  exo-setup  setup complete
-```
-
-
-## Booting up the application
-
-To test that everything works, let's check that the application boots up:
-
-```
-$ exo run
+>>> docker-compose --file run_development.yml up --build
+Creating network "todoapp_default" with the default driver
+Pulling exocom0.26.1 (originate/exocom:0.26.1)...
+0.26.1: Pulling from originate/exocom
+add3ddb21ede: Pull complete
+7b0e69015745: Pull complete
+Digest: sha256:56e176f25e21732ccad671375fba8177a04a3fc7ea8bc1decf9e715c6af961c6
+Status: Downloaded newer image for originate/exocom:0.26.1
+Building web-service
+Step 1/6 : FROM node
+latest: Pulling from library/node
+85b1f47fba49: Pull complete
+ba6bd283713a: Pull complete
+817c8cd48a09: Pull complete
+47cc0ed96dc3: Pull complete
+8888adcbd08b: Pull complete
+6f2de60646b9: Pull complete
+9dd205971dc0: Pull complete
+5859715a4691: Pull complete
+Digest: sha256:7c9099e0f68242387d7755eaa54c287e16cedd3cca423444ca773794f5f1e423
+Status: Downloaded newer image for node:latest
+ ---> c1d02ac1d9b4
+Step 2/6 : ENV NODE_ENV "development"
+ ---> Running in 61ad323a3101
+ ---> 0d4ade2ed6a6
+Removing intermediate container 61ad323a3101
+Step 3/6 : COPY package.json .
+ ---> 382f908e332a
+Step 4/6 : COPY yarn.lock .
+ ---> 2b2387775f86
+Step 5/6 : RUN yarn
+ ---> Running in e91437d97867
+yarn install v1.3.2
+[1/4] Resolving packages...
+[2/4] Fetching packages...
+info fsevents@1.1.2: The platform "linux" is incompatible with this module.
+info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
+[3/4] Linking dependencies...
+[4/4] Building fresh packages...
+Done in 10.40s.
+ ---> 714bc67a7dc4
+Removing intermediate container e91437d97867
+Step 6/6 : COPY . .
+ ---> 5a6ff5c8cff8
+Successfully built 5a6ff5c8cff8
+Successfully tagged todoapp_web-service:latest
+Creating exocom0.26.1 ...
+Creating exocom0.26.1 ... done
+Creating web-service ...
+Creating web-service ... done
+Attaching to exocom0.26.1, web-service
+exocom0.26.1    | ExoCom online at port 80
+exocom0.26.1    | 'web-service' registered
+web-service     | ExoRelay for 'web-service' online at port 80
+web-service     | HTML server online at port 3000
+web-service     | HTML server is running
 ```
 
-You should see output like:
+The Exosphere framework itself is written as a bunch of loosely coupled
+services. We see a number of them in action here:
 
-```
-Running Todo application 0.0.1
-
- exocom      online at port 8000
-html-server  Server running at port 3000
- exorun      'html-server' is running using exorelay port 8001
- exorun      all services online
- exocom      received routing setup
- exorun      application ready
-```
+* **exocom** is the command that runs Exosphere applications. It starts the
```

Comment:
I could use a little assistance here. I'm not sure if replacing exorun with exocom in this text block is actually correct. Is exorun just not used here?

Reply:
I think I found an appropriate change to address this. Of course, any and all feedback is welcome.